app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      include Wor::Paginate
+      def index
+        rents = Rent.where(['user_id = :uid', { uid: params[:user_id] }])
```

Comment:
I think we are adding a lot of complexity here, we can just use something like: 
`Model.where(column: params[:parameter])`

Check this out in this link: http://guides.rubyonrails.org/active_record_querying.html#equality-conditions